js/main.js

```diff
@@ -0,0 +1,131 @@
+const moduleBars = document.querySelector('#mobile-menu');
+const moduleOverlay = document.querySelector('.mobile-module-overlay');
+const moduleMenu = document.querySelector('.desktop-module-nav');
+const moduleMenuTimes = document.querySelector('.modal-1');
+
+moduleBars.addEventListener('click', () => {
+  moduleOverlay.style.display = 'block';
+  moduleMenu.style.display = 'block';
+});
+moduleMenuTimes.addEventListener('click', () => {
+  moduleOverlay.style.display = 'none';
+  moduleMenu.style.display = 'none';
+});
+moduleOverlay.addEventListener('click', () => {
+  moduleOverlay.style.display = 'none';
+  moduleMenu.style.display = 'none';
+});
+
+const speakers = [
+  {
+    id: 'speaker-1',
+    image1: './img/speaker-2.jpg',
+    names: 'Serkan Şehvelioğlu',
+    jobs: 'Php,mysql Software Developer in Muğla',
+    paragraph: 'Serkan Şehvelioğlu he have master degree and six years have experiment',
+  },
+  {
+    id: 'speaker-2',
+    image1: './img/speaker-1.jpg',
+    names: 'Berkay Dörtkardeş',
+    jobs: 'Php,mysql Software Developer in Muğla',
+    paragraph: 'Berkay Dörtkardeş he have master degree and six years have experiment',
+
+  },
+  {
+    id: 'speaker-2',
+    image1: './img/speaker-1.jpg',
+    names: 'Berkay Dörtkardeş',
+    jobs: 'Php,mysql Software Developer in Muğla',
+    paragraph: 'Berkay Dörtkardeş he have master degree and six years have experiment',
+
+  },
+  {
+    id: 'speaker-1',
+    image1: './img/speaker-2.jpg',
+    names: 'Serkan Şehvelioğlu',
+    jobs: 'Php,mysql Software Developer in Muğla',
+    paragraph: 'Serkan Şehvelioğlu he have master degree and six years have experiment',
+  },
+  {
+    id: 'speaker-1',
+    image1: './img/speaker-2.jpg',
+    names: 'Serkan Şehvelioğlu',
+    jobs: 'Php,mysql Software Developer in Muğla',
+    paragraph: 'Serkan Şehvelioğlu he have master degree and six years have experiment',
+  },
+  {
+    id: 'speaker-1',
+    image1: './img/speaker-1.jpg',
+    names: 'Serkan Şehvelioğlu',
+    jobs: 'Php,mysql Software Developer in Muğla',
+    paragraph: 'Serkan Şehvelioğlu he have master degree and six years have experiment',
+  },
+];
+
+window.addEventListener('load', () => {
+  if (window.innerWidth < 768) {
+    for (let index = 0; index < speakers.length; index += 1) {
+      if (index === 2) {
+        break;
+      } else {
+        const element = document.createElement('future-speakers');
+        element.innerHTML = `
+            <div class="speakers">
+            <div class="speaker-img">
+                <img src="${speakers[index].image1}" alt="">
```

Comment:
- [ ] OPTIONAL
It's a good practice to provide `alt` names for your images so as to improve the accessibility features of your site.  Kindly endeavor to add `alt` names to **all** your images.